wfe/wfe.go

```diff
@@ -431,23 +431,23 @@ func (wfe *WebFrontEndImpl) handleCertStatusBySerial(
 
 func (wfe *WebFrontEndImpl) Handler() http.Handler {
 	m := http.NewServeMux()
-	// GET only handlers
-	wfe.HandleFunc(m, DirectoryPath, wfe.Directory, "GET")
-	// Note for noncePath: "GET" also implies "HEAD"
-	wfe.HandleFunc(m, noncePath, wfe.Nonce, "GET")
+	// GET & POST handlers
+	wfe.HandleFunc(m, DirectoryPath, wfe.Directory, http.MethodGet, http.MethodPost)
```

Comment:
It looks like all the HandleFuncs now allow MethodPost. Maybe it's better to change the HandleFunc definition so it automatically adds POST? And then adds whatever methods are left in the varargs?

Reply:
I feel like that will make reading the code more confusing and we're unlikely to be adding new handlers with much frequency. Do you feel strongly about it? I have a light preference for the explicit version.

Reply:
I don't feel strongly about it; the explicit version is fine.